crates/ruff_python_ast/src/str.rs

```diff
@@ -67,6 +69,15 @@ pub fn is_triple_quote(content: &str) -> bool {
     TRIPLE_QUOTE_STR_PREFIXES.contains(&content) || TRIPLE_QUOTE_BYTE_PREFIXES.contains(&content)
 }
 
+/// Return `true` if the string expression is an implicit concatenation.
+pub fn is_implicit_concatenation(content: &str) -> bool {
```

Comment:
I think I prefer this interface at-present since it's consistent with the rest of this module, which operates directly on strings (and thus has no dependencies, isn't coupled to the AST, etc.).